Fuse.pretty/Fuseholder_Clip-5x20mm_Bel_FC-203-22_Lateral_P17.80x5.00mm_D1.17mm_Horizontal.kicad_mod

```diff
@@ -0,0 +1,50 @@
+(module Fuseholder_Clip-5x20mm_Bel_FC-203-22_Lateral_P17.80x5.00mm_D1.17mm_Horizontal (layer F.Cu) (tedit 5D30A842)
+  (descr "Fuseholder Clips, 5x20mm Cylinder Fuse, Pins Inline, Horizontal, Bel FC-203-22, https://www.belfuse.com/resources/datasheets/circuitprotection/ds-cp-0672-fuse-clips-series.pdf")
```

Comment:
Lateral at the description. Other dimensions are good.

Reply:
Fixed the description. I didn't put any current ratings in any of them; is it required?

Reply:
No it is not, just a thought that i think it is helpful..